pkg/reconciler/ingress/ingress.go

```diff
@@ -116,7 +117,7 @@ func (reconciler *Reconciler) updateIngress(ingress *v1alpha1.Ingress) error {
 		return err
 	}
 
-	ready, err := reconciler.statusManager.IsReady(ingress)
+	ready, err := reconciler.statusManager.IsReady(context.TODO(),ingress)
```

Comment:
What is this `context.TODO()` ?

Reply:
It requires a context. I think we don't have any we can use there, so.. I use context.TODO()

Reply:
:+1: